tests/main/upgrade-from-2.15/task.yaml

```diff
@@ -6,7 +6,11 @@ prepare: |
     dpkg --purge snapd
 
 restore: |
-    dpkg --purge ubuntu-core-launcher snap-confine
+    # Just remove when it was installed
+    if [ -e old_install_done ]; then
+        dpkg --purge snapd ubuntu-core-launcher snap-confine
+    fi
+    apt install -y "${GOHOME}"/snapd_*.deb
```

Comment:
Isn't this done in prepare suite each?

Reply:
@bboozzoo thanks for reviewing, snapd is just installed when the suite is prepared, then its state is restored in each suite preparation.